src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/service/TranscriptionWorkflowAuditTest.java

```diff
@@ -0,0 +1,82 @@
+import uk.gov.hmcts.darts.transcriptions.model.UpdateTranscriptionRequest;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.data.history.RevisionMetadata.RevisionType.INSERT;
+import static uk.gov.hmcts.darts.common.enums.SecurityRoleEnum.SUPER_ADMIN;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionUrgencyEnum.STANDARD;
+
+class TranscriptionWorkflowAuditTest extends IntegrationBase {
+
+    @Autowired
+    private TranscriptionService transcriptionService;
+
+    @Autowired
+    private GivenBuilder given;
+
+    @Test
+    void auditsWhenTranscriptionsAreCreated() {
+        given.anAuthenticatedUserWithGlobalAccessAndRole(SUPER_ADMIN);
+        var transcriptionRequestDetails = createTranscriptionRequestDetailsWithDefaults();
+
+        var createTranscriptionResponse = transcriptionService.saveTranscriptionRequest(transcriptionRequestDetails, true);
+
+        var transcriptionRevisions = dartsDatabase.findTranscriptionRevisionsFor(createTranscriptionResponse.getTranscriptionId());
+        assertThat(transcriptionRevisions.getLatestRevision().getMetadata().getRevisionType()).isEqualTo(INSERT);
+    }
+
+    @Test
+    void auditsWhenTranscriptionsWorkflowsAreTransitioned() {
+        var userAccountEntity = given.anAuthenticatedUserWithGlobalAccessAndRole(SUPER_ADMIN);
+        var transcriptionRequestDetails = createTranscriptionRequestDetailsWithDefaults();
+        var createTranscriptionResponse = transcriptionService.saveTranscriptionRequest(transcriptionRequestDetails, false);
+
+        transcriptionService.updateTranscriptionAdmin(
+            createTranscriptionResponse.getTranscriptionId(),
+            new UpdateTranscriptionRequest().transcriptionStatusId(7).workflowComment("new comment"),
+            true);
+
+        var auditActivity = findAuditActivity("Amend Transcription Workflow", dartsDatabase.findAudits());
+        assertThat(auditActivity.getUser().getId()).isEqualTo(userAccountEntity.getId());
+
+        var transcriptionWorkflowRevisions = dartsDatabase.findTranscriptionWorkflowRevisionsFor(createTranscriptionResponse.getTranscriptionId());
+        assertThat(transcriptionWorkflowRevisions.getLatestRevision().getMetadata().getRevisionType()).isEqualTo(INSERT);
+
+        var transcriptionCommentRevisions = dartsDatabase.findTranscriptionCommentRevisionsFor(createTranscriptionResponse.getTranscriptionId());
+        assertThat(transcriptionCommentRevisions.getLatestRevision().getMetadata().getRevisionType()).isEqualTo(INSERT);
+    }
+
+    private AuditEntity findAuditActivity(String activity, List<AuditEntity> audits) {
+        return audits.stream()
+            .filter(audit -> activity.equals(audit.getAuditActivity().getName()))
+            .findFirst().orElseThrow();
+    }
+
+    private TranscriptionRequestDetails createTranscriptionRequestDetailsWithDefaults() {
```

Comment:
could this be moved into one of the Test data classes ?

Reply:
That I'm aware of we don't currently have test data classes for api model data class, only entities.  I have been thinking about introducing something, I'm doing another auditing ticket so will look at introducing something then